source/clear-linux/tutorials/multi-boot/multi-boot-restore-bl.rst

```diff
@@ -14,43 +15,48 @@ longer the default boot loader. To restore it, follow the steps below.
 
    .. figure:: figures/multi-boot-restore-bl-1.png
 
-      Figure 1: |CL|: Console
+      Figure 1: |CL|: Console.
 
-#. Log in as *root*.
+#. Log in and get root privileges.
+
+   .. code-block:: bash
+
+      sudo -s
```

Comment:
Please keep the original "Log in as *root*."  Lines 20-24, as suggested, will work in this case.